recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - glue-core >=0.10
     - matplotlib
     - qtpy
-    - pyqt 4.11.*
+    - pyqt
```

Comment:
I would do `pyqt 5.6.*` instead to be on the safe. Otherwise users with different configurations may get  `defaults`
`qt > 5.6` that we may not know how it was build. (That can still happen at the micro version level or even at a build number level but it is probably less likely.)